src/ReadingBuddy.java

```diff
@@ -391,16 +490,24 @@ public static ArrayList<String[]> addNewUserData(String uname, String pw, ArrayL
         return allUsers;
     }
 
-    public static void updateBookBuddysEntry(BookGroup groupInfo){
+    //NEW SPRINT3
+    public static CurrentBook getSpecificCurrentBook(String userName, int bookID){
```

Comment:
in-code comment should be added saying: the intended function of this function is object retrieval of CurrentBook objects that are known to exist (better to create a different function to check for existence if needed in the future as to avoid the creation of invalid objects)